.github/workflows/0-everything.yml

```diff
@@ -306,6 +311,34 @@ jobs:
           -NvaUsername (ConvertTo-SecureString -String '${{secrets.NVA_USERNAME}}' -AsPlainText -Force) `
           -NvaPassword (ConvertTo-SecureString -String '${{secrets.NVA_PASSWORD}} '-AsPlainText -Force)
 
+  identity:
+    name: Identity
+    if: github.event.inputs.deployIdentity == 'true'
+
+    needs:
+      - Logging
+      - HubNetworking
+
+    runs-on: ubuntu-latest
+    
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v3
+
+    - name: Configure PowerShell modules
```

Comment:
Interesting! That will probably resolve the issue we were having before.

Reply:
That might be it. we can test it out after the merge.